x/programs/rust/examples/token/src/lib.rs

```diff
@@ -1,6 +1,13 @@
 use wasmlanche_sdk::{public, store::State, types::Address};
 
-/// Initializes the program with a name, symbol, and total supply.
+/// @title A basic ERC-20 compatible contract
+/// @author Ava Labs
+/// @notice Serves as a fungible token
+/// @dev Limits the token functionality to the bare minimum required
+
+/// @notice Initializes the program with a name, symbol, and total supply
+/// @param state The initial program state
+/// @return Whether the initialization succeeded
```

Comment:
This does not look good when generating docs
![image](https://github.com/ava-labs/hypersdk/assets/3286504/9adcbfcf-a9ee-4e3a-9946-ec39fa1888f5)

Reply:
Good point